modules/graceful-shutdown.adoc

```diff
@@ -38,42 +38,40 @@ $ oc -n openshift-kube-apiserver-operator get secret kube-apiserver-to-kubelet-s
 ----
 <1> To ensure that the cluster can restart gracefully, plan to restart it on or before the specified date. As the cluster restarts, the process might require you to manually approve the pending certificate signing requests (CSRs) to recover kubelet certificates.
 
-. Shut down all of the nodes in the cluster. You can do this from your cloud provider's web console, or run the following loop:
+. Mark the control plane nodes in the cluster as unschedulable. You can do this from your cloud provider's web console, or run the following loop:
```

Comment:
Indeed, from the following command, it not only mark the control plane nodes as unschedulable, but mark all the nodes (workers and control plane nodes). Please update the description correctly.

Reply:
Thanks. @lyman9966 . What would be the correct description?

Reply:
@lyman9966, if you aren't the right person to provide the description, can you point me to someone who could? Thanks!!

Reply:
@nalhadef I think the description should be: Mark all the nodes in the cluster as unschedulable. You can do this from your cloud provider's web console, or run the following loop:

Reply:
Hi, @nalhadef 
It seems keep not update. It should be: Mark all the nodes in the cluster as unschedulable.